### ISSUE TITLE
[General] Add CI for chart testing and deployment

### ISSUE THREAD

**Issue by Ornias1993** [gissilabs/charts#8]
Please add CI to check charts and deploy new releases. Relatively easy, like:
https://github.com/Skeen/helm-bitwarden_rs/tree/master/.github/workflows

**Comment by sgissi** [comment]
Good point, today I handle releases myself instead of using CI but I should give it a shot :)

**Comment by Ornias1993** [comment]
For a good example, feel free to take a look at mine:
https://github.com/Ornias1993/helm-charts
